Userland/Libraries/LibWeb/Loader/FrameLoader.cpp

```diff
@@ -398,6 +398,8 @@ void FrameLoader::resource_did_load()
     document->set_content_type(resource()->mime_type());
 
     browsing_context().set_active_document(document);
+    if (auto* page = browsing_context().page())
+        page->client().page_did_set_active_document(url);
```

Comment:
`set_active_document()` is called in more places...

Reply:
`page_did_create_(main_)document()` perhaps? also doesn't need the url argument anymore, btw.

Reply:
Changed it to `page_did_create_main_document()` and remove the argument.